ui/src/components/ApprovalsTracker/index.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect } from 'react';
+import * as Types from '@types';
+import * as Interfaces from '@interfaces';
+
+type Props = {
+    publicationData: Interfaces.Publication;
+    user: Types.UserType;
```

Comment:
'publicationData' prop includes everything we need to display on the approvals tracker
Do we really need the 'user' prop?